index.html

```diff
@@ -440,13 +440,13 @@ <h2 id="3Definitions">Definitions</h2>
 <!-- Maintain a fragment named "3alphaCompaction" to preserve incoming links to it -->
 <dfn id="3alphaCompaction"><dt>alpha compaction</dt></dfn>
 
-<dd>an implicit representation of transparent <a href=
-"#3pixel"><span class="Definition">pixels</span></a>. If every
+<dd>an implicit representation of transparent 
+<span class="Definition">[=pixels=]</span>. If every
```

Comment:
`span class="Definition">[=channels=]</span>` should be `<dfn>channels</dfn>` per respec, right?

Reply:
Elsewhere, there is <dfn>channel</dfn>. [=channel=] and [=channels=] then link to is.

The class "Definition" is used when linking to a definition in this case.

Strangely, I can't quite trace the legacy of this at the moment. It seems like even in the current, 2nd Edition of the spec there is no css class Definition.

Reply:
Ah, okay.
The Oct 1st, 1996 edition of the spec (later revised on May 20th, 2003) had a stylesheet, https://www.w3.org/TR/2003/PR-PNG-20030520/isostyle.css which defined span.Definition.

However, in the May 20th, 2003 revision that stylesheet is no longer referenced. So all of these are currently unused.

Reply:
.... so we can remove the `<span class="Definition">`?

Reply:
Also, I am really not a fan of mixing HTML and MD. If there is `<dfn>channel</dfn>` somewhere, then we can simply say `<a>channel</a>` instead of the obscure and respec-specific `[=pixels=]`.

Reply:
I believe we can remove `<span class="Definition">`.
If we do, that should be in a separate pull request / issue. And it isn't blocking anything so perhaps low priority.

Reply:
I misunderstood something.

I thought you were referring to `<a href="#fragment">blah</a>` and `[[=blah=]]`.
I'm reviewing your pull request now and I see `<a>blah</a>`, which is what you originally mentioned. I see ReSpec does reformat that the same way it would for `[[=blah=]]`.

Now that I better understand your question, I agree that using `<a>blah</a>` when we want ReSpec formatting is ideal.

Reply:
> However, in the May 20th, 2003 revision that stylesheet is no longer referenced. So all of these are currently unused.

Originally the isostyle style sheet was an "Alternate Stylesheet" i.e. one that the user could optionally decide to use. Over time, browsers de-emphasized, hid, or removed alternate style sheets. In particular, Chrome applies all the alternate stylesheets in addition to the main one.

This results in ugly green definitions everywhere, also fully-justified text which in early browsers was awful.

Given that the ISO house style forbids colored text, I don't even know what the isostyle.css is meant to be doing.